testing/utils/openshift_ci/pr_args.sh

```
@@ -93,7 +93,7 @@ REQUIRED_AUTHOR=$(jq -r .user.login <<< "$pr_json")
 REQUIRED_AUTHOR_ASSOCIATION=CONTRIBUTOR
 
 echo "# PR comments URL: $PR_COMMENTS_URL" >&2
-last_user_test_comment=$(echo "$last_comment_page_json" \
+last_user_test_comment=$(printf '%s' "$last_comment_page_json" \
```

Shellcheck (0.11.0)

[error] You can't have | between this redirection and the command it should apply to.

(SC2189)